packages/docs/content/components/button.mdx

```diff
@@ -11,172 +11,111 @@ medicare:
   sketchLink: Vr4yyWw
 ---
 
-import { Alert, Button, ArrowIcon, TextInput } from '@cmsgov/design-system';
+import { ArrowIcon, Button } from '@cmsgov/design-system';
 
-## Default Button
+## Examples
 
-<EmbeddedExample>
-  <div>
-    <Button className="ds-u-margin-right--1">Default</Button>
-    <Button className="ds-c-button--hover ds-u-margin-right--1">Hover</Button>
-    <Button className="ds-c-button--active ds-u-margin-right--1">Active</Button>
-    <Button className="ds-c-button--focus ds-u-margin-right--1">Focus</Button>
-    <Button disabled>Disabled</Button>
-  </div>
-</EmbeddedExample>
+The following tables give an overview of the different permutations of buttons available in the selected design system. The controlling traits are the named _variation_, whether it's the _main_ vs. _alternate_ color, and the _on-light_ vs. _on-dark_ context. Each of these traits has its own example later on showing how to apply it.
```

Comment:
I'm unsure about using the word "permutations", but I can't think of a good alternative. Ideas? Ideas on making this section clearer in general?

Reply:
Instead of "permutations", maybe something like "different available button styles"? Or update the paragraph a little like:

The following tables give an overview of the different available button styles in the selected design system. Each button has three controllable traits that can be used in various combinations. They are:
1. _Variation_: which affects the border and background styles
2. Color: defined as either _main_ or _alternate_
3. Context: specified as either _on-light_ or _on-dark_

See the below examples of how these are used.

Reply:
I like the rewrite!

Reply:
"different permutations" -> "variations"